lib/napa/middleware/logger.rb

```diff
@@ -38,16 +38,17 @@ def format_request(env)
           # do nothing, params is already set
         end
 
-        request_data = {
-          method:           request.request_method,
-          path:             request.path_info,
-          query:            filtered_query_string(request.query_string),
-          host:             Napa::Identity.hostname,
-          pid:              Napa::Identity.pid,
-          revision:         Napa::Identity.revision,
-          params:           filtered_parameters(params),
-          remote_ip:        request.ip
-        }
+        request_data = {}.tap do |h|
+          h[:method]      = request.request_method
+          h[:path]        = request.path_info
+          h[:query]       = filtered_query_string(request.query_string)
+          h[:host]        = Napa::Identity.hostname
+          h[:pid]         = Napa::Identity.pid
+          h[:revision]    = Napa::Identity.revision
+          h[:params]      = params unless ENV['SILENCE_REQUEST_PARAMS_LOG'] == 'true'
```

Comment:
Do you want to bypass the `filtered_parameters` here to show everything or did you have trouble with failing specs? For the latter see https://github.com/bellycard/napa/pull/258